content/v2/openapi.yml

```diff
@@ -4903,6 +4929,29 @@ components:
               - ns2.dnsimple.com
               - ns3.dnsimple.com
               - ns4.dnsimple.com
+    ZoneNsRecordsUpdate:
+      description: Zone name server attributes
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              ns_names:
```

Comment:
If you allow to pass sets, what's the purpose of allowing also individual name servers?

Reply:
The purpose is to be able to set a zone's NS records to values that aren't defined in a name server set. This way, you can add multiple name server sets and/or custom additional names. This matches the UI from the shape (and implementation thus far).